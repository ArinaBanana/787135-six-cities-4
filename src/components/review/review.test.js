import React from "react";
import renderer from "react-test-renderer";
import Review from "./review.jsx";

const review = {
  id: 1,
  username: `Jon`,
  rating: 4,
  message: `message`,
  date: `date`,
  userAvatar: `path`,
  isPro: true,
};

it(`Should render Review`, () => {
  const tree = renderer.create(<Review review={review} />).toJSON();

  expect(tree).toMatchSnapshot();
});
