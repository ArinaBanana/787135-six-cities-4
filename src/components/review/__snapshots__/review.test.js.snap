// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Review 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="path"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Jon
    </span>
    <span
      className="property__user-status"
    >
      Pro
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      message
    </p>
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      date
    </time>
  </div>
</li>
`;
